ui/src/app/pages/login/account-creation/account-creation.spec.tsx

```diff
@@ -123,8 +126,9 @@ it('should handle invalid Email', () => {
   expect(wrapper.exists('#invalidEmailError')).toBeFalsy();
 });
 
+// TODO remove after we switch to verified institutional affiliation
 it('should display Institution name and role option by default', () => {
-  serverConfigStore.next({...defaultConfig, enableNewAccountCreation: true});
+  serverConfigStore.next({...defaultConfig, enableNewAccountCreation: true, requireInstitutionalVerification: false});
   const wrapper = component();
   const institutionName = wrapper.find('[data-test-id="institutionname"]');
```

Comment:
aside: this `data-test-id` should actually be "institution-name".  I don't understand why this test passes.

Reply:
yes.  My suspicion is that `wrapper.find()`'s return value doesn't work how we are assuming it does.

Reply:
😡